docs/ProfilerOptions.md

```diff
@@ -0,0 +1,113 @@
+| `-l --lib`     | `lib`           | Prepend library names to symbols, e.g. ``libjvm.so`JVM_DefineClassWithSource``.                                                                                                                                                                                                                                                                                                                                                                      |
+| `--total`      | `total`         | Count the total value of the collected metric instead of the number of samples, e.g. total allocation size.                                                                                                                                                                                                                                                                                                                                          |
+
+## Options not applicable JFR, FlameGraph and Tree view outputs
+| asprof     | Launch as agent | Description                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+|------------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-a --ann` | `ann`           | Annotate JIT compiled methods with `_[j]`, inlined methods with `_[i]`, interpreted methods with `_[0]` and C1 compiled methods with `_[1]`.                                                                                                                                                                                                                                                                                                         |
+
+## Appendix
+
+### Event Option
+
+In allocation profiling mode the top frame of every call trace is the class
+of the allocated object, and the counter is the heap pressure (the total size
+of allocated TLABs or objects outside TLAB).
+
+In lock profiling mode the top frame is the class of lock/monitor, and
+the counter is number of nanoseconds it took to enter this lock/monitor.
+
+Two special event types are supported on Linux: hardware breakpoints
```

Comment:
There are more than two now: also PMU events, kprobes and uprobes

Reply:
Addressed in [e6282c8](https://github.com/async-profiler/async-profiler/pull/1029/commits/e6282c83bc0aa6007df310fd5e638e9d4d5ed6f1)